packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -26,7 +27,14 @@ const ETH_CHAIN_ID_MAPPINGS: Record<string, EthNetwork> = {
   'eip155:3': { network: 'ropsten', chain: 'ETH', chainId: 3, networkId: 3, type: 'Test' },
   'eip155:4': { network: 'rinkeby', chain: 'ETH', chainId: 4, networkId: 4, type: 'Test' },
   'eip155:5': { network: 'goerli', chain: 'ETH', chainId: 5, networkId: 5, type: 'Test' },
-  'eip155:100': { network: 'mainnet', chain: 'Gnosis', chainId: 100, networkId: 100, type: 'Test' },
+  'eip155:100': {
+    network: 'mainnet',
+    chain: 'Gnosis',
+    chainId: 100,
+    networkId: 100,
+    type: 'Test',
+    endpoint: 'https://rpc.ankr.com/gnosis',
```

Comment:
where did we find this endpoint URL?  Are we confident it's a good and stable one?

Reply:
I have used it before and it appears to be reliable re: https://chainlist.org/

Reply:
This was also suggested by Stefan of Gnosis

Reply:
Sounds good!